ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh

```diff
@@ -48,9 +48,10 @@ if openshift-install coreos print-stream-json 2>/tmp/err.txt >${SHARED_DIR}/core
    echo "Using stream metadata"
    ova_url=$(jq -r '.architectures.x86_64.artifacts.vmware.formats.ova.disk.location' < ${SHARED_DIR}/coreos.json)
 else
-  if ! grep -qF 'unknown command "coreos"' /tmp/err.txt; then
+  if ! grep -q 'unknown command \"coreos\"' /tmp/err.txt; then
```

Comment:
We need the `-F` here, otherwise grep interprets the `\`:

```
$ grep  'unknown command \"coreos\"' /tmp/err.txt
$ grep -F  'unknown command \"coreos\"' /tmp/err.txt
time="2021-04-01T18:01:26Z" level=fatal msg="Error executing openshift-install: unknown command \"coreos\" for \"openshift-install-4.7\""
$
```

Reply:
yep got the same result when doing it similar to the script.